spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/storage/queue/StorageQueueTemplate.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.integration.storage.queue;
+
+import com.microsoft.azure.spring.integration.core.Checkpointer;
+import com.microsoft.azure.spring.integration.core.Memoizer;
+import com.microsoft.azure.spring.integration.core.PartitionSupplier;
+import com.microsoft.azure.spring.integration.storage.queue.factory.StorageQueueFactory;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.queue.CloudQueue;
+import com.microsoft.azure.storage.queue.CloudQueueMessage;
+import org.springframework.util.Assert;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Function;
+
+public class StorageQueueTemplate implements StorageQueueOperation {
+    private final StorageQueueFactory storageQueueFactory;
+    private final Function<String, Checkpointer<CloudQueueMessage>> checkpointGetter =
+            Memoizer.memoize(this::createCheckpointer);
+    private int visibilityTimeoutInSeconds = 30;
```

Comment:
@Getter @Setter for this

Reply:
If use lombok annotation in tamplate, can not call get/set method by operation interface